vault-tool-server/test/test.hs

```diff
@@ -254,21 +274,71 @@ talkToVault addr = do
     secretId <- _VaultAppRoleSecretIdGenerateResponse_SecretId <$> vaultAppRoleSecretIdGenerate authConn "foo-role" ""
 
     arConn <- connectToVaultAppRole manager addr roleId secretId
-    throws (vaultRead arConn pathSmall :: IO (VaultSecretVersion FunStuff)) >>= (@?= True)
-
-    vaultSeal authConn
+    throws (KeyValueV2.vaultRead arConn pathSmall :: IO (KeyValueV2.VaultSecretVersion FunStuff)) >>= (@?= True)
 
-    status5 <- vaultSealStatus unauthConn
-    status5 @?= VaultSealStatus
-        { _VaultSealStatus_Sealed = True
-        , _VaultSealStatus_T = 2
-        , _VaultSealStatus_N = 4
-        , _VaultSealStatus_Progress = 0
+totpTests :: VaultConnection Authenticated -> IO ()
+totpTests authConn = do
+    vaultNewMount authConn "totp" VaultMount
```

Comment:
Is it worth making a helper function to define a TOTP mount?

Reply:
Are you suggesting a helper function in the test module or part of the TOTP library?

Reply:
I'm suggesting it for the TOTP library, but I don't know if there is a "common case" for it to assist.

Reply:
For our use case I don't think it makes sense. We will spin up vault and create all the mount points via ansible or something similar. So the service have some sort of config that specifies how to reach vault and what the TOTP mount point is. I'm not sure how other users of the library intend to create their mounts. So I'm inclined to hold off on this helper function at the moment.

Reply:
SGTM